messagebus/src/vespa/messagebus/messagebus.cpp

```diff
@@ -112,7 +112,7 @@ MessageBus::MessageBus(INetwork &net, const MessageBusParams &params) :
     _routingTables(),
     _sessions(),
     _protocolRepository(std::make_unique<ProtocolRepository>()),
-    _msn(std::make_unique<Messenger>()),
+    _msn(std::make_unique<Messenger>(true, true)),
```

Comment:
Is it currently safe to have thread-skipping be the default behavior for this constructor overload? Or should this be `false, false` instead?

Reply:
This constructor is only used by tests. My thought is that everything except distributor and proton should run with future defaults.